ci-operator/step-registry/baremetalds/assisted/operator/conf/baremetalds-assisted-operator-conf-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ baremetalds assisted operator conf command ************"
+
+cat << EOF > "${SHARED_DIR}/dev-scripts-additional-config"
```

Comment:
output does look quite empty
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/19470/rehearse-19470-pull-ci-openshift-assisted-service-ocm-2.3-e2e-metal-assisted-operator-ztp/1406658749870903296/artifacts/e2e-metal-assisted-operator-ztp/baremetalds-assisted-operator-conf/build-log.txt
in case it's bothering you

Reply:
It doesn't. The output stream is redirected to the file.
You can see the data exist when the config file is sourced
```bash
Using CONFIG /root/dev-scripts/config_root.sh
++(common.sh:38): source(): CONFIG=/root/dev-scripts/config_root.sh
++(common.sh:45): source(): source /root/dev-scripts/config_root.sh
+++(/root/dev-scripts/config_root.sh:1): source(): export OPENSHIFT_RELEASE_IMAGE=registry.build01.ci.openshift.org/ci-op-9kprq5mt/release@sha256:0b9a29c963ac0be59342d244368d44f3ab892da9c3e66caa84fbdb5a2ffee9ef
+++(/root/dev-scripts/config_root.sh:1): source(): OPENSHIFT_RELEASE_IMAGE=registry.build01.ci.openshift.org/ci-op-9kprq5mt/release@sha256:0b9a29c963ac0be59342d244368d44f3ab892da9c3e66caa84fbdb5a2ffee9ef
+++(/root/dev-scripts/config_root.sh:2): source(): export ADDN_DNS=147.75.207.207
+++(/root/dev-scripts/config_root.sh:2): source(): ADDN_DNS=147.75.207.207
+++(/root/dev-scripts/config_root.sh:3): source(): export OPENSHIFT_CI=true
+++(/root/dev-scripts/config_root.sh:3): source(): OPENSHIFT_CI=true
+++(/root/dev-scripts/config_root.sh:4): source(): export WORKER_MEMORY=16384
+++(/root/dev-scripts/config_root.sh:4): source(): WORKER_MEMORY=16384
+++(/root/dev-scripts/config_root.sh:5): source(): export ENABLE_LOCAL_REGISTRY=true
+++(/root/dev-scripts/config_root.sh:5): source(): ENABLE_LOCAL_REGISTRY=true
+++(/root/dev-scripts/config_root.sh:6): source(): export IP_STACK=v4
+++(/root/dev-scripts/config_root.sh:6): source(): IP_STACK=v4
+++(/root/dev-scripts/config_root.sh:7): source(): export NUM_EXTRA_WORKERS=1
+++(/root/dev-scripts/config_root.sh:7): source(): NUM_EXTRA_WORKERS=1
+++(/root/dev-scripts/config_root.sh:8): source(): export EXTRA_WORKER_VCPU=8
+++(/root/dev-scripts/config_root.sh:8): source(): EXTRA_WORKER_VCPU=8
+++(/root/dev-scripts/config_root.sh:9): source(): export EXTRA_WORKER_MEMORY=32768
+++(/root/dev-scripts/config_root.sh:9): source(): EXTRA_WORKER_MEMORY=32768
+++(/root/dev-scripts/config_root.sh:10): source(): export EXTRA_WORKER_DISK=120
+++(/root/dev-scripts/config_root.sh:10): source(): EXTRA_WORKER_DISK=120
+++(/root/dev-scripts/config_root.sh:11): source(): export PROVISIONING_NETWORK_PROFILE=Disabled
+++(/root/dev-scripts/config_root.sh:11): source(): PROVISIONING_NETWORK_PROFILE=Disabled
+++(/root/dev-scripts/config_root.sh:12): source(): export REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
+++(/root/dev-scripts/config_root.sh:12): source(): REDFISH_EMULATOR_IGNORE_BOOT_DEVICE=True
```

Reply:
I'm talking about verbosity, not functionality
But nevermind